javascript/modules/client/src/client.ts

```diff
@@ -17,7 +18,7 @@ import { BigNumber } from "@ethersproject/bignumber";
 import { AddressZero } from "@ethersproject/constants";
 import { ClientCore } from "./internal/core";
 import { DaoRole } from "./internal/interfaces/common";
-import { solidityPack } from "ethers/lib/utils";
+import { solidityPack, isAddress } from "ethers/lib/utils";
```

Comment:
I know you didn't introduce this, but could you fetch these functions from `@ethersproject/xxx`  instead?

Reply:
Heads up. I was unable to find the `solidityPack` method from `@ethersproject` for whatever reason. I've replaced this with the `pack` method from `@ehtersproject/solidity` which I believe is the same method.